examples/frontend/pages/cc/[project]/explore/index.tsx

```diff
@@ -31,7 +31,7 @@ const Explore = () => {
     )
 
     // hook that contains all the logic of explore page
-    const { items, nextPage } = useNFTExplore(
+    const { items, nextPage, notLogged } = useNFTExplore(
```

Comment:
I think in a new issue, we should add `useNFTExplore`  to `@cura/hooks` and try to improve it a bit.

Reply:
#206